src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/AssertionHelper.java

```diff
@@ -59,108 +67,112 @@ public static EncryptedAssertion anEidasEncryptedAssertion(String requestId,
+                .addEncryptedAssertion(anEidasEncryptedAssertionForTheTestRP(requestId, assertionIssuerId, aStubCountryEidasSignature(), attributeStatement))
                 .withSigningCredential(
                         new TestCredentialFactory(
                                 HUB_TEST_PUBLIC_SIGNING_CERT,
                                 HUB_TEST_PRIVATE_SIGNING_KEY
                         ).getSigningCredential());
     }
 
-    public static ResponseBuilder aValidEidasResponse(String requestId, String assertionIssuerId) {
-        return aValidEidasResponse(requestId, assertionIssuerId, anEidasAttributeStatement().build());
+    public static ResponseBuilder aValidEidasResponseSignedByStubCountryWithAnAssertionSignedByTheCountry(String requestId) {
+        return ResponseBuilder.aResponse()
+            .withId(requestId)
+            .withInResponseTo(requestId)
+            .withIssuer(anIssuer().withIssuerId(STUB_COUNTRY_ONE).build())
+            .addEncryptedAssertion(anEidasEncryptedAssertion(requestId, STUB_COUNTRY_ONE, aStubCountryEidasSignature()))
+            .withSigningCredential(
+                new TestCredentialFactory(
+                    STUB_COUNTRY_PUBLIC_PRIMARY_CERT,
+                    STUB_COUNTRY_PUBLIC_PRIMARY_PRIVATE_KEY
+                ).getSigningCredential());
     }
 
-    public static ResponseBuilder anInvalidSignatureEidasResponse(String requestId, String assertionIssuerId) {
+    public static ResponseBuilder aValidEidasResponseSignedByStubCountryWithAnAssertionSignedByTheCountry(String requestId, AttributeStatementBuilder attributeStatementBuilder) {
         return ResponseBuilder.aResponse()
-                .withId(requestId)
-                .withInResponseTo(requestId)
-                .withIssuer(anIssuer().withIssuerId(HUB_ENTITY_ID).build())
-                .addEncryptedAssertion(anEidasEncryptedAssertion(requestId, assertionIssuerId, anEidasSignature()))
-                .withSigningCredential(
-                        new TestCredentialFactory(
-                                STUB_COUNTRY_PUBLIC_PRIMARY_CERT,
-                                STUB_COUNTRY_PUBLIC_PRIMARY_PRIVATE_KEY
-                        ).getSigningCredential());
+            .withId(requestId)
+            .withInResponseTo(requestId)
+            .withIssuer(anIssuer().withIssuerId(STUB_COUNTRY_ONE).build())
+            .addEncryptedAssertion(anEidasEncryptedAssertionForTheTestRP(requestId, STUB_COUNTRY_ONE, aStubCountryEidasSignature(), attributeStatementBuilder.build()))
+            .withSigningCredential(
+                new TestCredentialFactory(
+                    STUB_COUNTRY_PUBLIC_PRIMARY_CERT,
+                    STUB_COUNTRY_PUBLIC_PRIMARY_PRIVATE_KEY
+                ).getSigningCredential());
     }
 
-    public static ResponseBuilder anInvalidAssertionSignatureEidasResponse(String requestId, String assertionIssuerId) {
+    @Deprecated
```

Comment:
Why `@Deprecated`?

Reply:
My mistake - I deprecated it to catch any old uses of it, as I'd replaced it.